packages/backend/bindings/capacitor/package.json

```diff
@@ -38,16 +38,16 @@
     "@capacitor/core": "^3.4.0"
   },
   "devDependencies": {
-    "@capacitor/android": "^3.4.0",
-    "@capacitor/core": "^3.4.0",
-    "@capacitor/ios": "^3.4.0",
+    "@capacitor/android": "^3.4.1",
+    "@capacitor/core": "^3.4.1",
+    "@capacitor/ios": "^3.4.1",
```

Comment:
The `yarn.lock` might be out of date after these changes

Reply:
thanks will update it